charts/crossplane-efs/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "1.1"
 description: "Helm Chart for installing a Custom Crossplane API, allowing to create an EFS Filesystem."
 name: crossplane-efs
-version: 1.0.0
+version: 1.1.0
 home: https://github.com/dvpe-cloud/dvpe-helm
```

Comment:
We should add the `deprecated: true` property here.

Documentation: https://helm.sh/docs/topics/charts/#deprecating-charts
Example: https://git.app.uib.no/caleno/helm-charts/-/blob/master/stable/nats/Chart.yaml#L6